src/main/java/ch/wisv/areafiftylan/products/model/TicketType.java

```diff
@@ -82,4 +82,48 @@ public TicketType(String name, String text, float price, int numberAvailable, Lo
     public void addPossibleOption(TicketOption option) {
         this.possibleOptions.add(option);
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Please document why this equals (and hashcode) is written manually, rather than using the one generated by Lombok. (You should be able to exclude/include some fields in the Lombok `@Data` annotation. Was that not sufficient?)

Reply:
Nice, didn't know that was possible